kernel/kernel-defaults/src/test/java/io/delta/kernel/defaults/client/TestDefaultJsonHandler.java

```diff
@@ -40,7 +42,10 @@
 import io.delta.kernel.types.StructType;
 import io.delta.kernel.utils.CloseableIterator;
 import io.delta.kernel.utils.Utils;
-import static io.delta.kernel.utils.DefaultKernelTestUtils.getTestResourceFilePath;
+
+import io.delta.kernel.defaults.utils.DefaultKernelTestUtils;
```

Comment:
nit: no new line? we can't actually enforce this I think

Reply:
I modified the import order in Intellij to remove the blank line.